mscore/resourceManager.cpp

```diff
@@ -45,6 +45,63 @@ ResourceManager::ResourceManager(QWidget *parent) :
       MuseScore::restoreGeometry(this);
       }
 
+//---------------------------------------------------------
+//   ExtensionFileSize
+//---------------------------------------------------------
+
+ExtensionFileSize::ExtensionFileSize(const int i)
+      : QTableWidgetItem(stringutils::convertFileSizeToHumanReadable(i))
+      , _size(i)
+      {}
+
+bool ExtensionFileSize::operator<(const QTableWidgetItem& nextItem) const
+      {
+      QString currentText = text();
+      QString nextText = nextItem.text();
+
+      QChar currentSizeType = currentText[currentText.size() - 2];
+      QChar nextSizeType = nextText[currentText.size() - 2];
+
+      // the integer before unit
+      int currentInt = currentText.split(QRegularExpression("\\s+"))[0].toInt();
+      int nextInt = nextText.split(QRegularExpression("\\s+"))[0].toInt();
+
+      return int2size(currentSizeType, currentInt) < int2size(nextSizeType, nextInt);
+      }
+
+//---------------------------------------------------------
+//   int2size
+//---------------------------------------------------------
+
+int ExtensionFileSize::int2size(QChar sizeType, int i)
+      {
+      if (sizeType == QChar(' ')) // the size is written as bytes
+            return i;
+      else if (sizeType == QChar('K')) // the size is written as kilobytes, and so on
+            return i * 0x400;
+      else if (sizeType == QChar('M'))
+            return i * 0x100000;
+      else if (sizeType == QChar('G'))
+            return i * 0x40000000;
+      else if (sizeType == QChar('T'))
+            return i * 0x10000000000;
+      }
+
+//---------------------------------------------------------
+//   LanguageFileSize
+//---------------------------------------------------------
+
+LanguageFileSize::LanguageFileSize(const double d)
+      : QTableWidgetItem(ResourceManager::tr("%1 KB").arg(d))
```

Comment:
Also is there any reason to change `kB` to `KB`? If there is then this PR should probably wait until 3.4 as this will require translations to be updated.

Reply:
It is not a change, but an addition (in this context and a removal in another) as far as I can see?
There's not much to translate about it though.
KB is 1024 bytes, kB is 1000 bytes, so we indeed need KB here

Reply:
3.4 is good, it's nice to have this done alongside #4833, which has something to do with the resource manager too.